test/BlockCipher/McryptTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace ZendTest\Crypt\BlockCipher;
+
+use Zend\Crypt\Symmetric;
+use Zend\Crypt\BlockCipher;
+
+class MCryptTest extends AbstractBlockCipherTest
+{
+    public function setUp()
+    {
+        try {
+            $this->cipher = new Symmetric\Mcrypt([
+                'algorithm' => 'aes',
+                'mode'      => 'cbc',
+                'padding'   => 'pkcs7'
+            ]);
+        } catch (Symmetric\Exception\RuntimeException $e) {
+            $this->markTestSkipped($e->getMessage());
```

Comment:
I'm wondering if we shouldn't fail the tests if there's a problem; it could be indicative of a regression in PHP or the mcrypt extension.